src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,123 @@
+package duke;
+
+import duke.command.*;
```

Comment:
Java coding standards states that imported classes should always be listed explicitly, although I can see why you would import * for this package _duke.command_. Should the different command classes be encapsulated such that _BYE_, _DELETE_ and _DONE_ can result in calling of a single factory method?